bindings/pyroot_experimental/PyROOT/python/ROOT/_facade.py

```diff
@@ -46,11 +46,21 @@ def __init__(self, module):
 
     def _fallback_getattr(self, name):
         # Try first in the global namespace, then in the ROOT namespace
-        # This allows to lookup e.g. ROOT.ROOT.Math as ROOT.Math
+        # and finally in gROOT (ROOT lists such as list of files,
+        # memory mapped files, functions, geometries ecc.)
+        # The first two attempts allows to lookup
+        # e.g. ROOT.ROOT.Math as ROOT.Math
         try:
             return getattr(gbl_namespace, name)
         except AttributeError:
-            return getattr(self.__dict__['ROOT_ns'], name)
+            try:
+                return getattr(self.__dict__['ROOT_ns'], name)
+            except AttributeError:
+                res = gROOT.FindObject(name)
+                if not res:
+                    raise AttributeError
```

Comment:
Do we need to add any message to this AttributeError raising? What shows on screen if we hit this case?